src/go/k8s/main.go

```diff
@@ -189,11 +193,10 @@ func main() {
 		storageAdvAddr = redpandacontrollers.DetermineAdvStorageAddr(storageAddr, setupLog)
 		storage := redpandacontrollers.MustInitStorage("/tmp", storageAdvAddr, 60*time.Second, 2, setupLog)
 
-		metricsH := helper.MustMakeMetrics(mgr)
+		// TODO this helper has gone away
+		// metricsH := helper.MustMakeMetrics(mgr)
```

Comment:
Is now something like `helper.NewMetrics(mgr, metrics.MustMakeRecorder(), v1.SourceFinalizer)`

Reply:
Thank you